design/nodes-machines-and-hosts.md

```diff
@@ -57,7 +57,7 @@ such as draining the node for maintenance. The definition of Node is
 owned by the Kubernetes core, and offers few opportunities to attach
 custom data.
 
-[Machine](https://kubernetes-sigs.github.io/cluster-api/common_code/machine_controller.html)
+[Machine](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine)
```

Comment:
I think it would be better to link to the concept docs here instead of the controller
```suggestion
[Machine](https://cluster-api.sigs.k8s.io/user/concepts#machine)
```